packages/core/__tests__/view/mixins/VertexMixin.test.ts

```diff
@@ -16,96 +16,140 @@ limitations under the License.
 
 import { describe, expect, test } from '@jest/globals';
 import { createGraphWithoutContainer } from '../../utils';
-import { type CellStyle, Geometry } from '../../../src';
+import { type Cell, type CellStyle, Geometry, type Graph } from '../../../src';
+
+const expectIsVertex = (cell: Cell, cellId: string) => {
+  expect(cell.getId()).toBe(cellId);
+  expect(cell.vertex).toBeTruthy();
+  expect(cell.edge).toBeFalsy();
+};
+
+const expectCellInModel = (graph: Graph, expectedCell: Cell, cellId: string) => {
```

Comment:
**suggestion:** no need to pass the `cellId`.
We already check that the id of the cell is equal to `cellId` in `expectIsVertex`.
The search of the cell in the model can be done by passing `cell.getId.`

Reply:
Done in e2be64ff